hardware/cpu/intel/xeon_e5_2623_3_ht.pan

```diff
@@ -0,0 +1,8 @@
+structure template hardware/cpu/intel/xeon_e5_2623_3_ht;
```

Comment:
What does this mean?

Reply:
Is this an e5 2623 v3 with hyperthreading enabled? That's a system setting, not a CPU setting.